container/lxd/client_test.go

```diff
@@ -28,7 +28,7 @@ func (s *connectionSuite) TestUpdateServerConfig(c *gc.C) {
 
 	updateReq := api.ServerPut{Config: map[string]interface{}{"key1": "val1"}}
 	gomock.InOrder(
-		cSvr.EXPECT().GetServer().Return(&api.Server{}, eTag, nil),
+		cSvr.EXPECT().GetServer().Return(&api.Server{}, eTag, nil).Times(2),
```

Comment:
Isn't this an error to do a round trip for server information 2 times? Shouldn't we get all the info we need on the first time?
Or is it that NewClient grabs it, but then we grab a fresh copy before we update ?

Reply:
Yes. NewClient now calls this before instantiating the client. It is used to set API extension flags.

Reply:
Yes. NewClient now calls this before instantiating the client. It is used to set API extension flags.